cmd/tailscale/netcheck.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main // import "tailscale.com/cmd/tailscale"
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"sort"
+	"time"
+
+	"github.com/pborman/getopt/v2"
+	"tailscale.com/netcheck"
+)
+
+func isSubcommand(cmd string) bool {
+	return len(getopt.Args()) == 1 && getopt.Args()[0] == cmd
+}
+
+func netcheckCmd() {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	report, err := netcheck.GetReport(ctx, log.Printf)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("\nReport:\n")
+	fmt.Printf("  * UDP: %v\n", report.UDP)
```

Comment:
You don't like tabs for cli indent?

(I don't have any useful opinions here.)

Reply:
Done.